### PR TITLE
[DEX-401] fix: algolia search usage

### PR DESCRIPTION
**Pull request by loicsay**

### CODE REVIEW

File: pkg/cmd/search/search.go [hunk 1]
```diff
@@ -51,8 +51,8 @@ func NewSearchCmd(f *cmdutil.Factory) *cobra.Command {
 			# Search for objects in the "BOOKS" index matching the query "tolkien" while setting the number of hits per page and specifying the page to retrieve
 			$ algolia search BOOKS --query "tolkien" --hitsPerPage 2 --page 4
 
-			# Search for objects in the "BOOKS" index matching the query "tolkien" and export the results to a new line delimited JSON (ndjson) file
-			$ algolia search BOOKS --query "tolkien" > books.ndjson
+			# Search for objects in the "BOOKS" index matching the query "tolkien" and export the results to a .json file
+			$ algolia search BOOKS --query "tolkien" > books.json
```
Comment (clemfromspace) [inline pkg/cmd/search/search.go:55]: Could we showcase how to use the `-o=jsonpath` output in order to export **only** the results to the `books.json` ?

Example: `-o="jsonpath={$.Hits}"`

WDYT?

**Review by clemfromspace — CHANGES_REQUESTED** [review]
One small suggestion 👍

File: pkg/cmd/search/search.go (hunk 1 above)
Comment (loicsay) [reply pkg/cmd/search/search.go:55]: Yes I agree, I tried to write something easier to understand, wdyt?

Comment (clemfromspace) [reply pkg/cmd/search/search.go:55]: Looks good 👍

**Review by clemfromspace — APPROVED** [review]

**Pull request merged**